src/frontend/parser.mly

```diff
@@ -426,7 +426,9 @@ common_expression:
        if
          List.length args = 1
          && ( String.is_suffix ~suffix:"_lpdf" id.name
-            || String.is_suffix ~suffix:"_lpmf" id.name )
+              || String.is_suffix ~suffix:"_lupdf" id.name
+              || String.is_suffix ~suffix:"_lpmf" id.name
+              || String.is_suffix ~suffix:"_lupf" id.name )
```

Comment:
This should be `_lupmf`

Reply:
Hmmm.. Iäm not certain whether my comment is showing for the correct line, but `_lupf` should be `_lupmf`

Reply:
Its showing correctly. I was sloppy. Thanks.